example/src/screens/HomeScreen.tsx

```diff
@@ -23,14 +23,10 @@ export const HomeScreen = () => {
   const [locationError, setLocationError] = useState("");
 
   useEffect(() => {
-    SitumPlugin.setConfiguration({
-      useRemoteConfig: true,
-    });
     registerCallbacks();
 
-    return () => {
```

Comment:
Why do you remove the code that stops positioning when the component is unmounted?